### ISSUE TITLE
How to generate certificates for custom subdomains ?

### ISSUE THREAD

**Issue by Neolex-Security** [yeswehack/pwn-machine#5]
Hello,
When I create a new folder like www.victim.com to make a subdomain like www.victim.com.mydomain.com, no certificate are generated and it use the default traefik certificate.
Is it supposed to generate one automatically, if not, is there a way to generate a certificate to not have to accept manually the certificate in the browser like it is done with the default subdomains ?
Thanks

**Comment by SakiiR** [comment]
You cannot use a wildcard certificate to match 3+ depth subdomains.

Let's encrypt does not accept that. What you might want to do instead is to add a directory `www-victim-com` to generate the subdomain `www-victim-com.mydomain.com`.

I hope I answered your question correctly (: 

TLDR:
- [x] `foo.mydomain.com` works
- [ ] `foo.bar.mydomain.com` doesn't work -> the certificate will not match this subdomain

**Comment by Neolex-Security** [comment]
@SakiiR Oh ok thank you, yes it works for 2 depth subdomains , I was wondering why it was working sometimes and sometimes not, sorry for the useless issue then.
Thanks !

**Comment by SakiiR** [comment]
A question/issue/suggestion is never useless 👍 

If someone comes here for the same issue, he will have an even quicker response than you had ! (less than 1sec google search instead of 3 days waiting for an issue response)
